l10n_cl_etd_account/models/account_invoice.py

```diff
@@ -6,6 +6,17 @@
 from odoo import api, models
 
 
+SII_MAPPING = {
+    # class_id.code: class_id.code of the refund
+    30: 60,  # Factura -> Nota de Crédito
+    33: 61,  # Factura Electrónica -> Nota de Crédito Electrónica
+    35: 60,  # Boleta -> Nota de Crédito
+    39: 61,  # Boleta Electrónica -> Nota de Crédito Electrónica
+    60: 55,  # Nota de Crédito -> Nota de Débito
+    61: 56,  # Nota de Crédito Electrónica -> Nota de Débito Electrónica
```

Comment:
This is good as a first approach, but the mapping could be done in the `sii.document-class`` Model.